perun-core/src/main/java/cz/metacentrum/perun/core/impl/MembersManagerImpl.java

```diff
@@ -147,6 +147,19 @@ public Member getMemberByUserId(PerunSession sess, Vo vo, int userId) throws Int
 		}
 	}
 
+	@Override
+	public List<Member> getMembersByUserWithStatus(PerunSession sess, User user, Status status) throws InternalErrorException {
+		try {
+			return jdbc.query("SELECT " + memberMappingSelectQuery + " FROM" +
+							" members WHERE members.user_id=? and members.status "+Compatibility.castToInteger()+"=?",
```

Comment:
Are you sure this is necessary?  I can't find casting to integer in any other method (using status) in this manager.

Reply:
I didn't want to end up with doubling the select based on DB type, like in ```getMembersCount()```. It is used like that in ```getGroupMembers()``` in ```GroupsManagerImpl```. The problem is, that jdbc postgres driver fails to convert passed number for the ```char(1)``` column, while oracle driver has no problem. But we could probably clean up all usages to always convert passed ```Status``` object to the string.

Anyway, I'll test it on devel, since I noticed whitespace before that cast and I'm not sure it won't fail the sql.